README.md

```diff
@@ -338,18 +338,20 @@ Prints out most recent the _StackDriver logs_. These are logs from `console.log`
 - `--open`: Open StackDriver logs in a browser.
 - `--setup`: Setup StackDriver logs.
 - `--watch`: Retrieves the newest logs every 5 seconds.
+- `--simplified`: Removes timestamps to the logs.
```

Comment:
Isn't the new default without timestamps?

Shouldn't this be `--timestamps: Adds timestamps to the logs.`?

Reply:
Without a major version bump, this is not possible. I've updated the terminology though since "Removes timestamps to the logs" makes no sense.